app/views/notifier/invite.markerb

```diff
@@ -1 +1,8 @@
-<%= t('.message', :invite_url => invite_code_url(@invitation_code), :diasporafoundation_url => 'https://diasporafoundation.org/', :diaspora_id => @inviter.try(:diaspora_handle)) %>
+<%= t('.message',
+  invite_url: invite_code_url(@invitation_code),
+  diasporafoundation_url: 'https://diasporafoundation.org/',
+  user: @inviter.try(:diaspora_handle) == @inviter.try(:name) ?
```

Comment:
After thinking about this, we should probably rename the existing `.name` to `.display_name` or something like this. And then we could create a new `.name` that returns `nil`. That would also be a good preparation for #3648.